db/seeders/20210106001140-users-seed.js

```diff
@@ -1,17 +1,21 @@
 'use strict';
 
+const bcrypt = require("bcrypt");
+const { secret } = require('../../config');
+
 module.exports = {
-  up: (queryInterface, Sequelize) => {
+  up: async (queryInterface, Sequelize) => {
```

Comment:
Async may not be needed here, no await in function.

Reply:
i am using await in this function to hash each user's passwords within the seed

Reply:
Yes I see the await to create the password hashes.

Reply:
Yikes, didn't see that...thanks for clarifying!